docs/support_files/api_reference.md

```diff
@@ -36,8 +36,9 @@ Defines a hook which is run after each scenario.
   * `tags`: string tag expression used to apply this hook to only specific scenarios. See [cucumber-tag-expressions](https://docs.cucumber.io/tag-expressions/) for more information
   * `timeout`: A hook-specific timeout, to override the default timeout.
 * `fn`: A function, defined as follows:
-  * The first argument will be an object of the form `{sourceLocation: {line, uri}, result: {duration, status}}` matching the event data for `test-case-finished`
+  * The first argument will be an object of the form `{sourceLocation: {line, uri}, result: {duration, status}, pickle: { tags: [{ name, location: { line, column } }], name, language, locations: [{ line, column }], steps: [{ text, arguments: [], locations: [{ line, column }] }]}}` matching the event data for `test-case-finished`
```

Comment:
Please remove the text: "matching the event data for `test-case-finished`" as that is no longer true. Same for the Before Hook.

Reply:
I removed that text. What is the event data for, and how come it no longer needs to match? I just want to understand what it is used for.